core/src/test/scala/org/apache/spark/sql/expression/ArithmeticAgg0Suite.scala

```diff
@@ -32,8 +32,11 @@ class ArithmeticAgg0Suite extends BaseTiSparkSuite {
     "select sum(tp_smallint) from full_data_type_table",
     "select sum(tp_float) from full_data_type_table",
     "select avg(tp_char) from full_data_type_table",
-    "select avg(tp_bigint) from full_data_type_table",
-    "select sum(tp_bigint) from full_data_type_table",
+    // Below two cases will causes overflow if no predicates applied
+    // While Spark does not check overflow for types other than decimal
+    // TiKV will do the check and throw exception
```

Comment:
`sum(tp_bigint)` overflow problem has been fixed in the latest master branch of TiKV, and TiKV will not throw overflow exception any more. I suggest removing the predicates and run these cases with the latest TiKV.